modules/dop/services/issue/adjust.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package issue
+
+import (
+	"time"
+)
+
+type condition func() bool
+
+type issueAdjuster interface {
+	planFinished(condition, *issueValidator) *time.Time
+}
+
+type issueCreateAdjuster struct{}
+
+func (i *issueCreateAdjuster) planFinished(match condition, v *issueValidator) *time.Time {
+	if !match() {
+		return nil
+	}
+	now := time.Date(time.Now().Year(), time.Now().Month(), time.Now().Day(), 0, 0, 0, 0, time.Now().Location())
```

Comment:
time.Now() to a temp variable, it call too many. each call will have some offset.

Reply:
done